### ISSUE TITLE
Freeforth fans, unite!

### ISSUE THREAD

**Issue by laheadle** [kevinfish/mcff#2]
Hello!

I'm a big FreeForth fan, and I wanted to say hello. I've been playing with FF and having tons of fun. I think it's a beautiful system, and very well documented. I'm using it to learn assembly AND forth at the same time, and I must say I feel like I am making great progress.

What are your plans for this repo? Do you have any news about FF in general? I tried to send a mail to Christophe but it bounced!

-Lyn

**Comment by kevinfish** [comment]
yes, I love it too.  My thoughts were to metacompile it to LLVM.  Not sure if it was to the source code or the bitcode yet.   That way it could go to almost anything in a pretty optimized form. 
Have you checked out LLVM yet?

-Kevin
 -----------
 Rise like lions after slumber    in unvanquishable number. shake your chains to earth like dew   which in sleep had fallen on you. ye are many   they are few. ~ Percy Bysshe Shelley 

```
On Monday, February 22, 2016 12:58 PM, Lyn Headley <notifications@github.com> wrote:
```

 Hello!I'm a big FreeForth fan, and I wanted to say hello. I've been playing with FF and having tons of fun. I think it's a beautiful system, and very well documented. I'm using it to learn assembly AND forth at the same time, and I must say I feel like I am making great progress.What are your plans for this repo? Do you have any news about FF in general? I tried to send a mail to Christophe but it bounced!-Lyn—

**Comment by laheadle** [comment]
No I haven't, but I'll be curious to see your ideas unfold. Good luck with metacompilation; that's something I don't fully understand yet. If you have any news or thoughts feel free to post an issue..I'm watching the repo!

**Comment by kevinfish** [comment]
googling you it appears you do web apps?   Have you checked out Hoplon, a ClojureScript Web Framework ?
|   |
|   |  |   |   |   |   |   |
| Hoplon, a ClojureScript Web FrameworkClojureScript Web Framework |
|  |
| View on hoplon.io | Preview by Yahoo |
|  |
|   |

 -Kevin ----------- Rise like lions after slumber    in unvanquishable number. shake your chains to earth like dew   which in sleep had fallen on you. ye are many   they are few. ~ Percy Bysshe Shelley 

```
On Tuesday, February 23, 2016 8:35 AM, Lyn Headley <notifications@github.com> wrote:
```

 No I haven't, but I'll be curious to see your ideas unfold. Good luck with metacompilation; that's something I don't fully understand yet. If you have any news or thoughts feel free to post an issue..I'm watching the repo!—

**Comment by laheadle** [comment]
I haven't seen it yet, but will check it out. Thanks!
